.github/workflows/ci.yml

```diff
@@ -0,0 +1,62 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+      - ci
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  install:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        include:
+          - plugin: "livesync"
+          - plugin: "payment_paypal"
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/cache@v2
+        id: cache-pip
+        with:
+          path: .venv
+          key: ${{ runner.os }}-pip-${{ hashFiles('requirements*.txt') }}
+      - name: Setup Python
+        uses: actions/setup-python@v1
+        if: steps.cache-pip.outputs.cache-hit != 'true'
+        with:
+          python-version: 3.9
+      - name: Get branch name (base)
+        if: github.event_name != 'pull_request'
+        shell: bash
+        run: echo "BRANCH_NAME=$(echo ${GITHUB_REF} | sed 's/.*\///')" >> $GITHUB_ENV
+      - name: Get branch name (pull request)
+        if: github.event_name == 'pull_request'
+        shell: bash
+        run: echo "BRANCH_NAME=$(echo ${GITHUB_HEAD_REF} | sed 's/.*\///')" >> $GITHUB_ENV
+      - name: Install python dependencies
+        if: steps.cache-pip.outputs.cache-hit != 'true'
+        run: |
+          python3 -m venv .venv
+          source .venv/bin/activate
+          pip install -U pip setuptools
+          pip install wheel
+          pip install -e git+https://github.com/indico/indico.git@${BRANCH_NAME}#egg=indico
```

Comment:
I think for testing we need to hardcode the branch here, since the push runs in your repo currently fail due to upstream not having a branch named `ci`.

Reply:
Yes, I agree. I made a few tests previously on a separate branch. But as soon as I finish we can run with `ci` once more.